src/pnpq/units.py

```diff
@@ -45,18 +65,66 @@ def to_mpc320_velocity(
     return rounded_velocity
 
 
-context.add_transformation(
+def mpc320_velocity_to_speed(
```

Comment:
[Velocity is what pint calls it by default](https://github.com/hgrecco/pint/blob/master/pint/default_en.txt#L242), let's not confuse things by calling it speed as well in the same line.

```suggestion
def mpc320_velocity_to_velocity(
```

is probably fine, or

```suggestion
def mpc320_velocity_to_pint_velocity(
```